aptos-move/framework/supra-framework/sources/vesting_without_staking.move

```diff
@@ -383,17 +383,19 @@ module supra_framework::vesting_without_staking {
     }
 
 	public entry fun vest_individual(contract_address: address, shareholder_address: address) acquires VestingContract {
-		assert_active_vesting_contract(contract_address);
```

Comment:
@sjoshisupra why we removed this check as the vesting contract can be terminated at the end of `vest_individual`. It will cause a problem when call vest method who will call `vest_individual` in a loop.